R/map_peptides_on_structure.R

```diff
@@ -9,12 +9,12 @@
 #'
 #' @param peptide_data a data frame that contains the input columns to this function. If structure
 #' or prediction files should be fetched automatically, please provide column names to the following
-#' arguments: **uniprot_id**, **pdb_id**, **chain**, **start_in_pdb**, **end_in_pdb**,
```

Comment:
I liked being able to simply provide start and end positions in different columns. That was usually the output of my data analysis pipeline.

Reply:
The problem is that start and end positions don't tell you anything of what is in between them. As I mention somewhere, the argument is not numeric but character. So your start position might be "50" and your end position "60", but in-between the positions are called "51", "51a", "51b" etc. This was unfortunately the case for some structures and resulted in errors. The bug in this case was to simplify it to start and end since this is just not correct. If you know what your start and end positions are and that in between it is just numerical you can just simply make a vector containing the positions and then collapse it with ";" as the separator. 
If the user works with standard structures, they don't need to do this since this is already done by the `find_peptide_in_structure()` function.